diff --git a/conf/distro/include/init-manager-openrc.inc b/conf/distro/include/init-manager-openrc.inc
index 3778748..5c4a58e 100644
--- a/conf/distro/include/init-manager-openrc.inc
+++ b/conf/distro/include/init-manager-openrc.inc
@@ -1,6 +1,6 @@
 # Use OpenRC for system initialization
 DISTRO_FEATURES:append = " openrc"
-DISTRO_FEATURES_BACKFILL_CONSIDERED:append = " sysvinit systemd"
+DISTRO_FEATURES_OPTED_OUT:append = " sysvinit systemd"
 VIRTUAL-RUNTIME_init_manager ??= "openrc-init"
 VIRTUAL-RUNTIME_initscripts ??= "openrc"
 VIRTUAL-RUNTIME_login_manager ??= "busybox"
diff --git a/conf/layer.conf b/conf/layer.conf
index 00fbbf7..4858fd5 100644
--- a/conf/layer.conf
+++ b/conf/layer.conf
@@ -33,4 +33,4 @@ BBFILE_PRIORITY_openrc-layer = "7"
 OPENRC_INITDIR = "${sysconfdir}/init.d"
 OPENRC_CONFDIR = "${sysconfdir}/conf.d"
 
-LAYERSERIES_COMPAT_openrc-layer = "whinlatter"
+LAYERSERIES_COMPAT_openrc-layer = "wrynose"
diff --git a/recipes-init/openrc-inittab-busybox/openrc-inittab-busybox_1.0.bb b/recipes-init/openrc-inittab-busybox/openrc-inittab-busybox_1.0.bb
index b96b213..460b040 100644
--- a/recipes-init/openrc-inittab-busybox/openrc-inittab-busybox_1.0.bb
+++ b/recipes-init/openrc-inittab-busybox/openrc-inittab-busybox_1.0.bb
@@ -3,7 +3,7 @@ LICENSE = "MIT"
 LIC_FILES_CHKSUM = "file://${COREBASE}/meta/COPYING.MIT;md5=3da9cfbcb788c80a0384361b4de20420"
 
 SRC_URI = "file://inittab.in"
-S = "${WORKDIR}/sources"
+S = "${UNPACKDIR}/sources"
 UNPACKDIR = "${S}"
 RPROVIDES:${PN} = "${@oe.utils.conditional('VIRTUAL-RUNTIME_init_manager', 'busybox', 'virtual-openrc-inittab', '', d)}"
 
diff --git a/recipes-init/openrc-inittab-sysvinit/openrc-inittab-sysvinit_1.0.bb b/recipes-init/openrc-inittab-sysvinit/openrc-inittab-sysvinit_1.0.bb
index ea05cc1..697019c 100644
--- a/recipes-init/openrc-inittab-sysvinit/openrc-inittab-sysvinit_1.0.bb
+++ b/recipes-init/openrc-inittab-sysvinit/openrc-inittab-sysvinit_1.0.bb
@@ -3,7 +3,7 @@ LICENSE = "MIT"
 LIC_FILES_CHKSUM = "file://${COREBASE}/meta/COPYING.MIT;md5=3da9cfbcb788c80a0384361b4de20420"
 
 SRC_URI = "file://inittab.in"
-S = "${WORKDIR}/sources"
+S = "${UNPACKDIR}/sources"
 UNPACKDIR = "${S}"
 RPROVIDES:${PN} = "${@oe.utils.conditional('VIRTUAL-RUNTIME_init_manager', 'sysvinit', 'virtual-openrc-inittab', '', d)}"
 
diff --git a/recipes-init/udev-init-scripts/udev-init-scripts_35.bb b/recipes-init/udev-init-scripts/udev-init-scripts_35.bb
index 9a78cf3..a222fef 100644
--- a/recipes-init/udev-init-scripts/udev-init-scripts_35.bb
+++ b/recipes-init/udev-init-scripts/udev-init-scripts_35.bb
@@ -6,7 +6,7 @@ LIC_FILES_CHKSUM = "file://init.d/udev;md5=14c7d7d983a7bc708cc23e88e7cdd856;begi
 SRC_URI = "https://gitweb.gentoo.org/proj/udev-gentoo-scripts.git/snapshot/udev-gentoo-scripts-${PV}.tar.gz"
 SRC_URI[sha256sum] = "51eef30ef99f7f184aa403d190c105c5565e48c1c2d35b1b9f9f052c099fe366"
 
-S = "${WORKDIR}/udev-gentoo-scripts-${PV}"
+S = "${UNPACKDIR}/udev-gentoo-scripts-${PV}"
 
 do_configure[noexec] = "1"
 
